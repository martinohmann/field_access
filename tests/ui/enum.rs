use field_access::FieldAccess;

#[derive(FieldAccess, Debug)]
pub enum Enum {
    A(i64),
    B(u8),
}

fn main() {}
