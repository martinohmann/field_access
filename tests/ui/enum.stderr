error: FieldAccess does not support enums
 --> tests/ui/enum.rs:4:1
  |
4 | / pub enum Enum {
5 | |     A(i64),
6 | |     B(u8),
7 | | }
  | |_^
